reppy/parser.py

```diff
@@ -110,9 +110,12 @@ def __init__(self, url, status, content, expires):
         if status == 200:
             self.parse(content)
         elif status in (401, 403):
-            logger.warn('Access disallowed to site %s (%i)' % (
+            logger.warn('Access forbidden to site %s (%i)' % (
                 url, status))
-            self.parse('''User-agent: *\nDisallow: /''')
+            if disallow_forbidden:
+                self.parse('''User-agent: *\nDisallow: /''')
+            else:
+                self.parse('')
```

Comment:
My thinking with this was I was attempting to be consistent with line 122 where self.parse gets called with an empty string. Does this method not need to be called?